index.html

```diff
@@ -0,0 +1,213 @@
+                        </li>
+                    </ul>
+                </div>
+                <div class="bg">
+                    <ul class="values-reasons" id="values">
+                        <li>
+                            <img
+                                src="/first-capstone-project/assets/images/responsive.png" />
+                            <li><p>Responsive</p></li>
+                            <li>
+                                <p>To the needs of the children in our target
+                                    group
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
+
+                <div class="bg">
+                    <ul class="values-reasons">
+                        <li>
+                            <img
+                                src="/first-capstone-project/assets/images/enriching.png" />
+                            <li><p>Enriching</p></li>
+                            <li>
+                                <p>
+                                    bettering lives through education and
+                                    other avenues
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
+
+            </div>
+            <button type="button" class="donate-btn mobile">Donate</button>
+            <a href="/first-capstone-project/pages/about.html"
+                class="desktop see-program">See
+                the whole program</a>
+
+            <div class="donate">
+                <img src="/first-capstone-project/assets/images/donate.png" />
+            </div>
+
+        </section>
+        <section class="projects">
+            <h3>Upcoming Events</h3>
+            <hr class="title-hr" />
+            <div class="events-list"></div>
+            <div class="more-btn" id="more-btn">
+                <p class="more">MORE</p>
+                <i class="arrow down"></i>
+
+            </div>
+            <div class="more-btn less-btn" id="less-btn">
+                <p class="more">LESS</p>
+                <i class="arrow up"></i>
+
+            </div>
+        </section>
+        <section class="partners">
+            <h3>Partners</h3>
+            <hr class="title-hr">
+            <div class="partners-desktop">
+                <div class="partners-logo">
+                    <img src="/first-capstone-project/assets/images/google.svg"
+                        alt="Google" />
+                    <img src="/first-capstone-project/assets/images/mozilla.svg"
+                        alt="Mozilla" class="mozilla" />
+                    <img src="/first-capstone-project/assets/images/airbnb.png"
+                        alt="Google" />
+
+                </div>
+                <div class="partners-logo">
+                    <div class="naver">
+                        <img
```

Comment:
- [ OPTIONAL ] Please on mobile view, You should not have the footer of the home page as you can see in the design below
    
    ![image](https://github.com/NoelLincoln/first-capstone-project/assets/92303891/64c0ed48-7634-48bc-977e-aef0072cd96a)